app/kuma-dp/pkg/dataplane/metrics/server.go

```diff
@@ -30,6 +30,23 @@ var (
 var (
 	prometheusRequestHeaders = []string{"accept", "accept-encoding", "user-agent", "x-prometheus-scrape-timeout-seconds"}
 	logger                   = core.Log.WithName("metrics-hijacker")
+
+	// holds prometheus content types in order of priority.
+	prometheusPriorityContentType = []expfmt.Format{
+		expfmt.FmtOpenMetrics_1_0_0,
```

Comment:
Have I understood correctly that each format is a superset of the next?

Reply:
The formats are kept/sorted as per their priority. So, Yes, each format is a superset of the next.